docs/manual-installation.md

```diff
@@ -6,56 +6,112 @@ There are four components which need to be deployed in order to run Antrea:
 
 * The OpenVSwitch daemons `ovs-vswitchd` and `ovsdb-server`
 
+* The controller `antrea-controller`
+
 * The agent `antrea-agent`
 
 * The CNI plugin `antrea-cni`
 
-* **Optional** The controller `antrea-controller`
-
 ## Instructions
 
 ### OpenVSwitch
 
 Open vSwitch >= 2.8.0 userspace daemon `ovs-vswitchd` and `ovsdb-server` should run on all worker nodes. See
 [Installing Open vSwitch](https://docs.openvswitch.org/en/latest/intro/install/#installation-from-packages) for details.
 
+### antrea-controller
+
+`antrea-controller` is required to implement Kubernetes Network Policies. At any time, there should be only a single
+active replica of `antrea-controller`.
+
+1. Grant the `antrea-controller` ServiceAccount necessary permissions to Kubernetes APIs. You can apply
+[controller-rbac.yaml](/build/yamls/base/controller-rbac.yml) to do it.
+```shell script
+kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/base/controller-rbac.yml
```

Comment:
Not 100% sure but should we assume that the target audience has cloned the repo and use the relative path (from the root of the repo) here: `build/yamls/base/controller-rbac.yml`? After all if they have built the binaries manually (agent / controller) they must have cloned the repo. That only really matters if they are making changes to the YAML files though.

Reply:
Just trying to keep the command simply to run with less assumption and prerequisites. 
No strong opinion, I will change if you think local relative path is better.